huaweicloud/resource_huaweicloud_cdm_cluster_v1_test.go

```diff
@@ -40,23 +42,34 @@ func TestAccCdmClusterV1_basic(t *testing.T) {
 	})
 }
 
-func testAccCdmClusterV1_basic(val string) string {
+func testAccCdmClusterV1_basic(rName string) string {
 	return fmt.Sprintf(`
-resource "huaweicloud_networking_secgroup_v2" "secgroup" {
-  name = "terraform_test_security_group%s"
-  description = "terraform security group acceptance test"
+data "huaweicloud_availability_zones" "test" {}
+
+data "huaweicloud_vpc" "test" {
+  name = "vpc-default"
+}
+
+data "huaweicloud_vpc_subnet" "test" {
+  name = "subnet-default"
 }
 
-resource "huaweicloud_cdm_cluster_v1" "cluster" {
-  availability_zone = "%s"
-  flavor_id = "a79fd5ae-1833-448a-88e8-3ea2b913e1f6"
-  name = "terraform_test_cdm_cluster%s"
-  security_group_id = "${huaweicloud_networking_secgroup_v2.secgroup.id}"
-  subnet_id = "%s"
-  vpc_id = "%s"
-  version = "1.8.5"
```

Comment:
does the "1.8.5" version still works?

Reply:
It may not work. I used the 1.8.5 version to test, an error occurred. The test was successful after changing version to 2.8.4 according to the console.